src/Features/CSharp/Portable/Completion/KeywordRecommenders/AbstractKeywordRecommender.cs

```diff
@@ -24,6 +24,18 @@ internal class AbstractKeywordRecommender : AbstractSyntacticSingleKeywordRecomm
             SyntaxKind.OverrideKeyword,
         };
```

Comment:
Do we also need to adjust `ImplicitKeywordRecommender` or `ExplicitKeywordRecommender`?

Reply:
> Do we also need to adjust ImplicitKeywordRecommender or ExplicitKeywordRecommender?

I didn't notice any difference in behavior around these keywords in interface vs. in class.

Reply:
I was looking at the code and those two recommenders use `ClassStructRecordTypeDeclarations`.
That suggests that we're currently not offering `implicit` or `explicit` keywords in interfaces. But I think that we should now offer them.

Reply:
But I will double check, not under umbrella of this PR though

Reply:
> I was looking at the code and those two recommenders use ClassStructRecordTypeDeclarations.
That suggests that we're currently not offering implicit or explicit keywords in interfaces. But I think that we should now offer them.

 I will double check, not under umbrella of this PR though